source/octf/analytics/statistics/FilesystemStatistics.h

```diff
@@ -77,6 +79,12 @@ class FilesystemStatistics {
 
     uint64_t m_devId;
     uint64_t m_partId;
+
+    using StatisticsCase = proto::FilesystemStatisticsEntry::NameCase;
```

Comment:
Isn't proto file missing in this pull request?

Reply:
It's not, each oneof has corresponding auto-generated enumerator